### ISSUE TITLE
FluxStore 1.6.6b - unable to build - error in font_awesome_flutter

### ISSUE THREAD

**Issue by Mohammad-AlZabibi** [inspireui/support#2635]
I'm trying to build the project but i get this error 

Target debug_android_application failed: FileSystemException: Cannot copy file to 'C:\Users\MHD\projects\codecanyon-qOfaVirs-fluxstore-the-universal-woocommerce-magento-opencart-mobile-app-by-flutter-framework\FluxStore 1.6.6b\source\build\app\intermediates\flutter\debug\flutter_assets\packages\font_awesome_flutter\lib\fonts\fa-brands-400.ttf', path = 'C:\Users\MHD\Downloads\Compressed\flutter_windows_v1.0.0-stable\flutter\.pub-cache\hosted\pub.dartlang.org\font_awesome_flutter-8.8.1\lib\fonts\fa-brands-400.ttf' (OS Error: The system cannot find the path specified.
, errno = 3)
build failed.

This my flutter version 
Flutter 1.12.13+hotfix.9 • channel stable • https://github.com/flutter/flutter.git
Framework • revision f139b11009 (10 days ago) • 2020-03-30 13:57:30 -0700
Engine • revision af51afceb8
Tools • Dart 2.7.2

**Comment by minhcasi** [comment]
@philipvu30 - please help to investigate this issue

**Comment by philipvu30** [comment]
Hi,

You can try to remove this 3 files 
![image](https://user-images.githubusercontent.com/54442328/78967558-04a6b380-7b2d-11ea-85e8-5e389f3af4bc.png)

and run these commands:
- flutter clean
- flutter packages get

**Comment by Mohammad-AlZabibi** [comment]
Hello, Thanks for responding 

I've tried this solution but still getting the same error.  

----------------------------------------------------------------

Target debug_android_application failed: FileSystemException: Cannot copy file to 'C:\Users\MHD\projects\codecanyon-qOfaVirs-fluxstore-the-universal-woocommerce-magento-opencart-mobile-app-by-flutter-framework\FluxStore 1.6.6b\source\build\app\intermediates\flutter\debug\flutter_assets\packages\font_awesome_flutter\lib\fonts\fa-brands-400.ttf', path = 'C:\Users\MHD\Downloads\Compressed\flutter_windows_v1.0.0-stable\flutter\.pub-cache\hosted\pub.dartlang.org\font_awesome_flutter-8.8.1\lib\fonts\fa-brands-400.ttf' (OS Error: The system cannot find the path specified.
, errno = 3)
build failed.

FAILURE: Build failed with an exception.

* Where:
Script 'C:\Users\MHD\Downloads\Compressed\flutter_windows_v1.0.0-stable\flutter\packages\flutter_tools\gradle\flutter.gradle' line: 780

* What went wrong:
Execution failed for task ':app:compileFlutterBuildDebug'.
> Process 'command 'C:\Users\MHD\Downloads\Compressed\flutter_windows_v1.0.0-stable\flutter\bin\flutter.bat'' finished with non-zero exit value 1

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 32s
Finished with error: Gradle task assembleDebug failed with exit code 1


----------------------------------------------------------------
This flutter doctor -v 

[√] Flutter (Channel stable, v1.12.13+hotfix.9, on Microsoft Windows [Version 10.0.18363.752], locale en-US)
    • Flutter version 1.12.13+hotfix.9 at C:\Users\MHD\Downloads\Compressed\flutter_windows_v1.0.0-stable\flutter
    • Framework revision f139b11009 (11 days ago), 2020-03-30 13:57:30 -0700
    • Engine revision af51afceb8
    • Dart version 2.7.2


[√] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
    • Android SDK at C:\Users\MHD\AppData\Local\Android\Sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-29, build-tools 28.0.3
    • ANDROID_HOME = C:\Users\MHD\AppData\Local\Android\Sdk
    • Java binary at: C:\Program Files\Android\Android Studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 1.8.0_212-release-1586-b04)
    • All Android licenses accepted.

[√] Android Studio (version 3.6)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin version 45.0.1
    • Dart plugin version 192.7761
    • Java version OpenJDK Runtime Environment (build 1.8.0_212-release-1586-b04)

[√] VS Code (version 1.41.1)
    • VS Code at C:\Users\MHD\AppData\Local\Programs\Microsoft VS Code
    • Flutter extension version 3.7.1

[√] Connected device (1 available)
    • MI 8 Pro • ac735fbf • android-arm64 • Android 10 (API 29)

• No issues found!


--------------------------------------------------------

Any other solutions ?

**Comment by Mohammad-AlZabibi** [comment]
@philipvu30  please any other solutions ?

**Comment by Mohammad-AlZabibi** [comment]
@minhcasi  please?

**Comment by philipvu30** [comment]
Hi @Mohammad-AlZabibi ,

Can I have a look of the file pubspec.yaml?

**Comment by Mohammad-AlZabibi** [comment]
Hi @philipvu30 ,

This is my pubspac.yaml file : 
----------------------------------------

name: fstore
description: Mobile commerce app by Flutter

# The following defines the version and build number for your application.
# A version number is three numbers separated by dots, like 1.2.43
# followed by an optional build number separated by a +.
# Both the version and the builder number may be overridden in flutter
# build by specifying --build-name and --build-number, respectively.
# In Android, build-name is used as versionName while build-number used as versionCode.
# Read more about Android versioning at https://developer.android.com/studio/publish/versioning
# In iOS, build-name is used as CFBundleShortVersionString while build-number used as CFBundleVersion.
# Read more about iOS versioning at
# https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html
version: 1.6.6+3001

environment:
  sdk: ">=2.2.2 <3.0.0"

dependencies:
  logs:
    git: https://github.com/pq/logs
  flutter:
    sdk: flutter
  http: 0.12.0+4
  html_unescape: 1.0.1+3
  provider: 4.0.4
  after_layout: 1.0.7+2
  font_awesome_flutter: 8.8.1
  path: 1.6.4
  vector_math: 2.0.8
  carousel_pro: 1.0.0
  flutter_widget_from_html: 0.3.2+2
  configurable_expansion_tile: 1.0.0
  timeago: 2.0.26
  share: 0.6.3+6
  validate: 1.7.0
  country_pickers: 1.2.0
  shared_preferences: 0.5.6+3
  firebase_messaging: 6.0.12
  firebase_analytics: 5.0.11
  transparent_image: 1.0.0
  pull_to_refresh: 1.5.8
  localstorage: 3.0.1+4
  notification_permissions: 0.4.4
  flare_splash_screen: 3.0.1
  rate_my_app: 0.6.0+2
  page_indicator: 0.3.0
  extended_image: 0.7.2
  intro_slider: 2.2.9
  url_launcher: 5.4.2
  firebase_core: 0.4.4 # keep these version as there is new issue
  firebase_auth: 0.15.3 # with the latest firebase https://github.com/FirebaseExtended/flutterfire/issues/2084
  flutter_webview_plugin: 0.3.10+1
  connectivity: 0.4.8+2
  random_string: 2.0.1
  google_maps_flutter: 0.5.25+3
  place_picker: 0.9.11
  cached_network_image: 2.0.0
  image_picker: 0.6.4
  uuid: 2.0.4
  uni_links: 0.2.0
  flutter_staggered_grid_view: 0.3.0
  device_apps: 1.0.9
  auto_size_text: 2.1.0
  flutter_swiper: 1.1.6
  flutter_widgets: 0.1.12
  video_player: 0.10.8+1
  flutter_spinkit: 4.1.2+1
  percent_indicator: 2.1.1+1
  animated_text_kit: 2.0.0
  wave: 0.0.8
  http_auth: 0.2.9
  firebase_admob: 0.9.1
  facebook_audience_network: 0.5.0
  pin_code_fields: 2.4.0
  flutter_facebook_login: 3.0.0
  webview_flutter: 0.3.19+9
  rubber: 0.4.0
  flutter_localizations:
    sdk: flutter
  custom_splash: 0.0.2
  apple_sign_in: 0.1.0
  google_sign_in: 4.4.0
  google_fonts: 0.3.10
  flutter_statusbarcolor: 0.2.3
  app_settings: 3.0.1
  animated_splash: 1.0.0
  firebase_database: 3.1.3
  event_bus: 1.1.1
  country_code_picker: 1.3.7
  flutter_typeahead: 1.8.0
  flash: 1.2.3

dev_dependencies:
  flutter_launcher_icons: 0.7.4
  flutter_test:
    sdk: flutter
  pedantic: 1.8.0+1 # Analysis

# Run this script to generate the app icon: flutter pub run flutter_launcher_icons:main
flutter_icons:
  android: "launcher_icon"
  ios: true
  image_path: "assets/images/app_icon.png"

# For information on the generic Dart part of this file, see the
# following page: https://www.dartlang.org/tools/pub/pubspec

# The following section is specific to Flutter.
flutter:
  uses-material-design: true
  assets:
    - lib/config/
    - lib/config/mocks/
    - lib/config/states/
    - assets/icons/categories/
    - assets/icons/tabs/
    - assets/icons/profile/
    - assets/icons/payment/
    - assets/images/
    - assets/images/country/
    - assets/images/favicon/
    - assets/html/
    - assets/images/background/
  fonts:
    - family: MaterialIcons
      fonts:
        - asset: assets/fonts/MaterialIcons.ttf
    - family: Entypo
      fonts:
        - asset: assets/fonts/Entypo.ttf
    - family: FontAwesome
      fonts:
        - asset: assets/fonts/FontAwesome.ttf
    - family: MaterialCommunityIcons
      fonts:
        - asset: assets/fonts/MaterialCommunityIcons.ttf
    - family: Feather
      fonts:
        - asset: assets/fonts/Feather.ttf

  # An image asset can refer to one or more resolution-specific "variants", see
  # https://flutter.io/assets-and-images/#resolution-aware.

  # For details regarding adding assets from package dependencies, see
  # https://flutter.io/assets-and-images/#from-packages

  # To add custom fonts to your application, add a fonts section here,
  # in this "flutter" section. Each entry in this list should have a
  # "family" key with the font family name, and a "fonts" key with a
  # list giving the asset and other descriptors for the font. For
  # example:
  #
  # For details regarding fonts from package dependencies,
  # see https://flutter.io/custom-fonts/#from-packages
flutter_intl:
  enabled: true


------------------------------------------

**Comment by Mohammad-AlZabibi** [comment]
Hi @philipvu30 ,

This is my pubspec.yaml file : 
---------------


name: fstore
description: Mobile commerce app by Flutter

# The following defines the version and build number for your application.
# A version number is three numbers separated by dots, like 1.2.43
# followed by an optional build number separated by a +.
# Both the version and the builder number may be overridden in flutter
# build by specifying --build-name and --build-number, respectively.
# In Android, build-name is used as versionName while build-number used as versionCode.
# Read more about Android versioning at https://developer.android.com/studio/publish/versioning
# In iOS, build-name is used as CFBundleShortVersionString while build-number used as CFBundleVersion.
# Read more about iOS versioning at
# https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html
version: 1.6.6+3001

environment:
  sdk: ">=2.2.2 <3.0.0"

dependencies:
  logs:
    git: https://github.com/pq/logs
  flutter:
    sdk: flutter
  http: 0.12.0+4
  html_unescape: 1.0.1+3
  provider: 4.0.4
  after_layout: 1.0.7+2
  font_awesome_flutter: 8.8.1
  path: 1.6.4
  vector_math: 2.0.8
  carousel_pro: 1.0.0
  flutter_widget_from_html: 0.3.2+2
  configurable_expansion_tile: 1.0.0
  timeago: 2.0.26
  share: 0.6.3+6
  validate: 1.7.0
  country_pickers: 1.2.0
  shared_preferences: 0.5.6+3
  firebase_messaging: 6.0.12
  firebase_analytics: 5.0.11
  transparent_image: 1.0.0
  pull_to_refresh: 1.5.8
  localstorage: 3.0.1+4
  notification_permissions: 0.4.4
  flare_splash_screen: 3.0.1
  rate_my_app: 0.6.0+2
  page_indicator: 0.3.0
  extended_image: 0.7.2
  intro_slider: 2.2.9
  url_launcher: 5.4.2
  firebase_core: 0.4.4 # keep these version as there is new issue
  firebase_auth: 0.15.3 # with the latest firebase https://github.com/FirebaseExtended/flutterfire/issues/2084
  flutter_webview_plugin: 0.3.10+1
  connectivity: 0.4.8+2
  random_string: 2.0.1
  google_maps_flutter: 0.5.25+3
  place_picker: 0.9.11
  cached_network_image: 2.0.0
  image_picker: 0.6.4
  uuid: 2.0.4
  uni_links: 0.2.0
  flutter_staggered_grid_view: 0.3.0
  device_apps: 1.0.9
  auto_size_text: 2.1.0
  flutter_swiper: 1.1.6
  flutter_widgets: 0.1.12
  video_player: 0.10.8+1
  flutter_spinkit: 4.1.2+1
  percent_indicator: 2.1.1+1
  animated_text_kit: 2.0.0
  wave: 0.0.8
  http_auth: 0.2.9
  firebase_admob: 0.9.1
  facebook_audience_network: 0.5.0
  pin_code_fields: 2.4.0
  flutter_facebook_login: 3.0.0
  webview_flutter: 0.3.19+9
  rubber: 0.4.0
  flutter_localizations:
    sdk: flutter
  custom_splash: 0.0.2
  apple_sign_in: 0.1.0
  google_sign_in: 4.4.0
  google_fonts: 0.3.10
  flutter_statusbarcolor: 0.2.3
  app_settings: 3.0.1
  animated_splash: 1.0.0
  firebase_database: 3.1.3
  event_bus: 1.1.1
  country_code_picker: 1.3.7
  flutter_typeahead: 1.8.0
  flash: 1.2.3

dev_dependencies:
  flutter_launcher_icons: 0.7.4
  flutter_test:
    sdk: flutter
  pedantic: 1.8.0+1 # Analysis

# Run this script to generate the app icon: flutter pub run flutter_launcher_icons:main
flutter_icons:
  android: "launcher_icon"
  ios: true
  image_path: "assets/images/app_icon.png"

# For information on the generic Dart part of this file, see the
# following page: https://www.dartlang.org/tools/pub/pubspec

# The following section is specific to Flutter.
flutter:
  uses-material-design: true
  assets:
    - lib/config/
    - lib/config/mocks/
    - lib/config/states/
    - assets/icons/categories/
    - assets/icons/tabs/
    - assets/icons/profile/
    - assets/icons/payment/
    - assets/images/
    - assets/images/country/
    - assets/images/favicon/
    - assets/html/
    - assets/images/background/
  fonts:
    - family: MaterialIcons
      fonts:
        - asset: assets/fonts/MaterialIcons.ttf
    - family: Entypo
      fonts:
        - asset: assets/fonts/Entypo.ttf
    - family: FontAwesome
      fonts:
        - asset: assets/fonts/FontAwesome.ttf
    - family: MaterialCommunityIcons
      fonts:
        - asset: assets/fonts/MaterialCommunityIcons.ttf
    - family: Feather
      fonts:
        - asset: assets/fonts/Feather.ttf

  # An image asset can refer to one or more resolution-specific "variants", see
  # https://flutter.io/assets-and-images/#resolution-aware.

  # For details regarding adding assets from package dependencies, see
  # https://flutter.io/assets-and-images/#from-packages

  # To add custom fonts to your application, add a fonts section here,
  # in this "flutter" section. Each entry in this list should have a
  # "family" key with the font family name, and a "fonts" key with a
  # list giving the asset and other descriptors for the font. For
  # example:
  #
  # For details regarding fonts from package dependencies,
  # see https://flutter.io/custom-fonts/#from-packages
flutter_intl:
  enabled: true

**Comment by philipvu30** [comment]
Hi @Mohammad-AlZabibi ,

Please run command "flutter clean" and send your project to this email: toan@inspireui.com
Need to figure what is wrong with the source code. Thanks

**Comment by Mohammad-AlZabibi** [comment]
@philipvu30  
Okay , I've sent it, 
waiting for respond 
Thank you .

**Comment by philipvu30** [comment]
Received

**Comment by philipvu30** [comment]
Hi,

Where do you put your flutter folder in?

Is it in C:\src\flutter?

Please move the flutter folder to the above path and link it like this in Android Studio Settings. Then try it again.

![image](https://user-images.githubusercontent.com/54442328/79188979-c110ba00-7e4a-11ea-80b0-8da7d72354a1.png)

**Comment by Mohammad-AlZabibi** [comment]
Hi,

I've done that but still having the same problem, Any way i built the code
on mac then transfer the code to my Windows without flutter clean then it
works,

Now i have a working version, so you can close the issue if no one getting
the same problem.

Thanks for trying to help , and i'm ready for any experiments.

best regards.

Mohammad Al-Zabibi

[image: image.png]

**Comment by Kenpo84** [comment]
Hello, Im getting the bellow error when I run Fluxstore Vendor 1.6.6b. I really need help pls!!!
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, v1.12.13+hotfix.9, on Mac OS X 10.15.3 19D76,
    locale en-GH)
 
[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
[✓] Xcode - develop for iOS and macOS (Xcode 11.4.1)
[✓] Android Studio (version 3.6)
[✓] VS Code (version 1.44.1)
[✓] Connected device (1 available)

• No issues found!
kenpo@Kens-MBP ~ % 

Launching lib/main.dart on iPhone 11 Pro Max in debug mode...
Running pod install...
Running Xcode build...
Xcode build done.                                           39.5s
Failed to build iOS app
Error output from Xcode build:
↳
    ** BUILD FAILED **


Xcode's output:
↳
    /Users/kenpo/flutter/.pub-cache/hosted/pub.dartlang.org/uni_links-0.2.0/ios/Classes/UniLinksPlugin.m:6:31: error: cannot find protocol declaration for 'FlutterStreamHandler'
    @interface UniLinksPlugin () <FlutterStreamHandler>
                                  ^
    /Users/kenpo/flutter/.pub-cache/hosted/pub.dartlang.org/uni_links-0.2.0/ios/Classes/UniLinksPlugin.m:6:12: error: cannot find interface declaration for 'UniLinksPlugin'
    @interface UniLinksPlugin () <FlutterStreamHandler>
               ^
    /Users/kenpo/flutter/.pub-cache/hosted/pub.dartlang.org/uni_links-0.2.0/ios/Classes/UniLinksPlugin.m:6:12: error: class extension has no primary class
    /Users/kenpo/flutter/.pub-cache/hosted/pub.dartlang.org/uni_links-0.2.0/ios/Classes/UniLinksPlugin.m:6:12: error: class extension has no primary class
    /Users/kenpo/flutter/.pub-cache/hosted/pub.dartlang.org/uni_links-0.2.0/ios/Classes/UniLinksPlugin.m:11:17: warning: cannot find interface declaration for 'UniLinksPlugin'
    @implementation UniLinksPlugin {
                    ^
    /Users/kenpo/flutter/.pub-cache/hosted/pub.dartlang.org/uni_links-0.2.0/ios/Classes/UniLinksPlugin.m:12:3: error: unknown type name 'FlutterEventSink'
      FlutterEventSink _eventSink;
      ^
    /Users/kenpo/flutter/.pub-cache/hosted/pub.dartlang.org/uni_links-0.2.0/ios/Classes/UniLinksPlugin.m:24:41: error: no type or protocol named 'FlutterPluginRegistrar'
    + (void)registerWithRegistrar:(NSObject<FlutterPluginRegistrar> *)registrar {
                                            ^
    /Users/kenpo/flutter/.pub-cache/hosted/pub.dartlang.org/uni_links-0.2.0/ios/Classes/UniLinksPlugin.m:78:27: error: expected a type
    - (void)handleMethodCall:(FlutterMethodCall *)call result:(FlutterResult)result {
                              ^
    /Users/kenpo/flutter/.pub-cache/hosted/pub.dartlang.org/uni_links-0.2.0/ios/Classes/UniLinksPlugin.m:78:60: error: expected a type
    - (void)handleMethodCall:(FlutterMethodCall *)call result:(FlutterResult)result {
                                                               ^
    /Users/kenpo/flutter/.pub-cache/hosted/pub.dartlang.org/uni_links-0.2.0/ios/Classes/UniLinksPlugin.m:88:4: error: expected a type
    - (FlutterError *_Nullable)onListenWithArguments:(id _Nullable)arguments
       ^
    /Users/kenpo/flutter/.pub-cache/hosted/pub.dartlang.org/uni_links-0.2.0/ios/Classes/UniLinksPlugin.m:89:59: error: expected ')'
                                           eventSink:(nonnull FlutterEventSink)eventSink {
                                                              ^
    /Users/kenpo/flutter/.pub-cache/hosted/pub.dartlang.org/uni_links-0.2.0/ios/Classes/UniLinksPlugin.m:89:50: note: to match this '('
                                           eventSink:(nonnull FlutterEventSink)eventSink {
                                                     ^
    /Users/kenpo/flutter/.pub-cache/hosted/pub.dartlang.org/uni_links-0.2.0/ios/Classes/UniLinksPlugin.m:94:4: error: expected a type
    - (FlutterError *_Nullable)onCancelWithArguments:(id _Nullable)arguments {
       ^
    /Users/kenpo/flutter/.pub-cache/hosted/pub.dartlang.org/uni_links-0.2.0/ios/Classes/UniLinksPlugin.m:19:34: error: no known class method for selector 'alloc'
        _instance = [[UniLinksPlugin alloc] init];
                                     ^~~~~
    /Users/kenpo/flutter/.pub-cache/hosted/pub.dartlang.org/uni_links-0.2.0/ios/Classes/UniLinksPlugin.m:27:3: error: use of undeclared identifier 'FlutterMethodChannel'
      FlutterMethodChannel *channel =
      ^
    /Users/kenpo/flutter/.pub-cache/hosted/pub.dartlang.org/uni_links-0.2.0/ios/Classes/UniLinksPlugin.m:27:25: error: use of undeclared identifier 'channel'
      FlutterMethodChannel *channel =
                            ^
    /Users/kenpo/flutter/.pub-cache/hosted/pub.dartlang.org/uni_links-0.2.0/ios/Classes/UniLinksPlugin.m:28:8: error: use of undeclared identifier 'FlutterMethodChannel'
          [FlutterMethodChannel methodChannelWithName:kMessagesChannel
           ^
    /Users/kenpo/flutter/.pub-cache/hosted/pub.dartlang.org/uni_links-0.2.0/ios/Classes/UniLinksPlugin.m:30:14: error: no visible @interface for 'NSObject' declares the selector 'addMethodCallDelegate:channel:'
      [registrar addMethodCallDelegate:instance channel:channel];
       ~~~~~~~~~ ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    /Users/kenpo/flutter/.pub-cache/hosted/pub.dartlang.org/uni_links-0.2.0/ios/Classes/UniLinksPlugin.m:32:3: error: use of undeclared identifier 'FlutterEventChannel'
      FlutterEventChannel *chargingChannel =
      ^
    /Users/kenpo/flutter/.pub-cache/hosted/pub.dartlang.org/uni_links-0.2.0/ios/Classes/UniLinksPlugin.m:32:24: error: use of undeclared identifier 'chargingChannel'
      FlutterEventChannel *chargingChannel =
                           ^
    /Users/kenpo/flutter/.pub-cache/hosted/pub.dartlang.org/uni_links-0.2.0/ios/Classes/UniLinksPlugin.m:33:8: error: use of undeclared identifier 'FlutterEventChannel'
          [FlutterEventChannel eventChannelWithName:kEventsChannel
           ^
    fatal error: too many errors emitted, stopping now [-ferror-limit=]
    1 warning and 20 errors generated.
    Command CompileSwift failed with a nonzero exit code
    note: Using new build system
    note: Building targets in parallel
    note: Planning build
    note: Constructing build description
    warning: Mapping architecture armv7 to i386. Ensure that this target's Architectures and Valid Architectures build settings are configured correctly for the iOS Simulator platform. (in target 'image_picker' from project 'Pods')
    warning: Mapping architecture arm64 to x86_64. Ensure that this target's Architectures and Valid Architectures build settings are configured correctly for the iOS Simulator platform. (in target 'image_picker' from project 'Pods')

Could not build the application for the simulator.
Error launching application on iPhone 11 Pro Max.
